calour/plotting.py

```diff
@@ -517,25 +525,36 @@ def plot_scatter_matrix(exp: Experiment, field, feature_ids, title_field=None,
 
     Parameters
     ----------
-    field : str
+    fields : str
```

Comment:
I think it should be 'field' and not 'fields'? since it contains the name of only one column (i.e. usually i'd expect fields to be a list of column names)?

Reply:
nope. plan to make it do multi-task prediction (predict multi labels together). will finish it in next pr. this change doesn't impact current function.